cabal-install/main/Main.hs

```diff
@@ -522,8 +550,8 @@ replAction (replFlags, buildExFlags) extraArgs globalFlags = do
       -- be done to support sandboxes.
       _ <-
         reconfigure configureAction
-        verbosity distPref useSandbox noAddSource NoFlag
-        mempty [] globalFlags config
+          verbosity distPref useSandbox noAddSource NoFlag
+          mempty [] globalFlags config
```

Comment:
Unrelated whitespace change, just leave the code as is. This doesn't belong in this PR.